tests/console/postgresql_server.pm

```diff
@@ -36,5 +36,15 @@ sub run {
     destroy_pgsqldb;
 }
 
+sub post_fail_hook {
+    my ($self) = @_;
+    select_console 'log-console';
+    $self->SUPER::post_fail_hook;
```

Comment:
just curious - when you doing this , does it mean that SUPER::post_fail_hook will be actually called twice ?

Reply:
no. What the `sub post_fail_hook` is doing is overwriting the parent method so it would not be called at all. Hence it's called explicitly here